accepted/2022/simplify-output-paths.md

```diff
@@ -0,0 +1,87 @@
+# Simplify .NET Output Paths
+
+We'd like to simplify the output paths for .NET 8.
+
+Currently, the default output path for a .NET project includes folders for the `TargetFramework` and the `RuntimeIdentifier` (if it is set).  Additionally, the publish output goes in a `publish` subfolder of the output folder.  The resulting paths are as follows:
+
+- `bin\<Configuration>\<TargetFramework>` - Build output with no `RuntimeIdentifier`
+- `bin\<Configuration>\<TargetFramework>\<RuntimeIdentifier>` - Build output with `RuntimeIdentifier`
+- `bin\<Configuration>\<TargetFramework>\publish` - Publish output with no `RuntimeIdentifier`
+- `bin\<Configuration>\<TargetFramework>\<RuntimeIdentifier>\publish` - Publish output with `RuntimeIdentifier`
+
+This is rather complicated and inconsistent, which can make for a poor first impression of the .NET platform.
```

Comment:
Do we have any data to back up this claim? Does the existing paths, as they exist today, block adoption of dotnet? Do existing users of dotnet have issues with these paths?

Having data to show these points would be a great way to justify the rest of this work.

Reply:
I agree that this claim is subjective.  One form of data that could back it up would be to compare to other platforms.

Reply:
See my response [here](https://github.com/dotnet/designs/pull/281#issuecomment-1415979133) about our plan to get feedback, and that we are not going to turn this on by default to start with.